raiden-dapp/src/components/AmountInput.vue

```diff
@@ -225,20 +228,23 @@ $header-vertical-margin-mobile: 2rem;
           padding-left: 10px;
         }
       }
+
       &__message {
         color: $secondary-text-color;
         line-height: 1.1;
       }
     }
   }
+
   &__token-symbol {
     font-family: $main-font;
     color: $text-color;
     font-weight: 500;
     font-size: 14px;
     line-height: 21px;
-    text-align: center;
+    text-align: cente
```

Comment:
Intended? :thinking:

Reply:
Shit, there is something wrong with my editor. It behaves super strange.

Reply:
Okay apparently GH notified me about a wrong version. But the history of this line is funny. :laughing: